src/components/Router/index.js

```diff
@@ -92,8 +92,17 @@ class CrossDeviceMobileRouter extends Component {
     }
   }
 
-  setConfig = (actions) => (data) => {
-    const {token, steps, language, documentType, step: userStepIndex,clientStepIndex, woopraCookie} = data
+  setMobileConfig = (actions) => (data) => {
+    const {
+      token,
+      steps,
+      language,
+      documentType,
```

Comment:
it will be interesting to double check with @joninvski to make sure backend wise we are good with this change.

Reply:
Yes, good point. Will check with him if this will play nice with backend.

Reply:
Just checked it. We are ok. We just blacklist some fields:

https://gitlab.eu-west-1.mgmt.onfido.xyz/onfido/sdks/cross_device_sync/blob/3279fb2c90cbc62fe66033bb64def40786c0c680/src/message_handle.js#L113

And none of the blacklisted names are there:

https://gitlab.eu-west-1.mgmt.onfido.xyz/onfido/sdks/cross_device_sync/blob/3279fb2c90cbc62fe66033bb64def40786c0c680/src/message_handle.js#L17

So :+1: from the backend side